cockroachdb/Chart.yaml

```diff
@@ -2,8 +2,8 @@
 apiVersion: v1
 name: cockroachdb
 home: https://www.cockroachlabs.com
-version: 10.0.3
-appVersion: 22.2.3
+version: 10.0.5
```

Comment:
I intentionally bumped version from 10.0.3 to 10.0.5 (in case there were already any 10.0.4 in the wild)